src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -144,6 +144,16 @@
      */
     private int port;
 
+    /**
+     * The war file to be used instead of the one in the path or war/target/jenkins.war if it is core build.
+     */
+    private File war;
+
+    /**
+     * If true, all plugins from pom will be skipped. Extra plugins are not ignored.
+     */
+    private boolean skipAllPlugins = false;
```

Comment:
isn't this changing the original default behaviour because of https://github.com/jenkinsci/jenkins-test-harness/pull/415/files#diff-ade203da1dc447eff46cf045f9a21d8ac019ccffdae26359a6838a54a1ac48fbR283 ? Right now the `RealJenkinsRule` is executing all that in this PR is within the `if (skipAllPlugins)`. If by default `skipAllPlugins` is false, then that code won't be executed unless the method `skipAllPlugins` is called with `true` as parameter